deploy/runtime/help/html/battery_dispatch_fom.htm

```diff
@@ -47,7 +47,7 @@ <h3 class="p_Heading3_atocs_" style="page-break-inside: avoid; page-break-after:
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Initial state of charge, %</span></p>
 <p class="p_VariableDescription"><span class="f_VariableDescription">The state of charge of the battery at the beginning of the simulation. A value of 100% would be for a fully charged battery, 0% would be for a fully discharged battery.</span></p>
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Minimum time at charge state, min</span></p>
-<p class="p_VariableDescription"><span class="f_VariableDescription">This setting only applies to subhourly simulations. You can ignore it for hourly simulations. For sub-hourly simulations, there may be periods of time where the photovoltaic or other power system output varies above and below the load causing rapid cycling of the battery. This kind of cycling, especially if the cycles are deep, may degrade battery performance over time. The minimum time at charge state prevents the battery from changing between charging and discharging within the number of minutes that you specify.</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">This setting only applies to subhourly simulations. You can ignore it for hourly simulations. For sub-hourly simulations, there may be periods of time where the photovoltaic or other power system output varies above and below the load causing rapid cycling of the battery. Rapid cycling may also result for dispatch options that respond to power prices when prices change rapidly from time step to time step. This kind of cycling, especially if the cycles are deep, may degrade battery performance over time. The minimum time at charge state prevents the battery from changing between charging and discharging within the number of minutes that you specify.</span></p>
 <h3 class="p_Heading3_atocs_" style="page-break-inside: avoid; page-break-after: avoid; border-top: none; border-right: none; border-left: none;"><span class="f_Heading3_atocs_">Storage Dispatch Controller (Front of Meter)</span></h3>
 <p class="p_Text"><span class="f_Text">The battery dispatch options determine when the battery charges and discharges. The charge options determine any limits on how the battery can charge or discharge.</span></p>
```

Comment:
Additional comment on this file: no summary for "dispatch to custom time series." Does it need one?

You could copy the "input battery power targets" text from BTM and it would be accurate.

![image](https://user-images.githubusercontent.com/5530592/144282559-c59f099d-61fb-4744-8079-3a0b431cee54.png)